examples/aptos/README.md

```diff
@@ -2,27 +2,31 @@
 
 ## Prerequisite
 
-1. Build the Aptos modules
+1. Install Aptos in your local machine
+
+Download here: https://aptos.dev/cli-tools/aptos-cli-tool/
+
+Note: Our examples are tested on Aptos version `1.0.4`
+
+2. Build the Aptos modules
 
 ```bash
 npm ci
 npm run build-aptos
 ```
 
-2. Run the `aptos` local network
+3. Run the `aptos` local network
 
 ```bash
 aptos node run-local-testnet --with-faucet
 ```
 
-3. Run the `evm` local network
+4. Run the local server
 
 ```bash
-npm scripts/start
+npm run start
```

Comment:
Error
```
$ npm run start

> axelar-examples@1.0.0 start
> node scripts/runStart

Error: ENOENT: no such file or directory, open 'node_modules/@axelar-network/axelar-cgp-aptos/aptos/modules/axelar/build/AxelarFramework/package-metadata.bcs'
    at Object.openSync (node:fs:585:3)
    at Object.readFileSync (node:fs:453:35)
    at AptosNetwork.<anonymous> (/home/stephen/workspace/axelar/axelar-examples/node_modules/@axelar-network/axelar-local-dev/dist/aptos/AptosNetwork.js:64:50)
    at Generator.next (<anonymous>)
    at /home/stephen/workspace/axelar/axelar-examples/node_modules/@axelar-network/axelar-local-dev/dist/aptos/AptosNetwork.js:8:71
    at new Promise (<anonymous>)
    at __awaiter (/home/stephen/workspace/axelar/axelar-examples/node_modules/@axelar-network/axelar-local-dev/dist/aptos/AptosNetwork.js:4:12)
    at AptosNetwork.deploy (/home/stephen/workspace/axelar/axelar-examples/node_modules/@axelar-network/axelar-local-dev/dist/aptos/AptosNetwork.js:63:16)
    at AptosNetwork.deployAxelarFrameworkModules (/home/stephen/workspace/axelar/axelar-examples/node_modules/@axelar-network/axelar-local-dev/dist/aptos/AptosNetwork.js:96:21)
    at /home/stephen/workspace/axelar/axelar-examples/node_modules/@axelar-network/axelar-local-dev/dist/aptos/aptosNetworkUtils.js:27:46 {
  errno: -2,
  syscall: 'open',
  code: 'ENOENT',
  path: 'node_modules/@axelar-network/axelar-cgp-aptos/aptos/modules/axelar/build/AxelarFramework/package-metadata.bcs'
}
Skip Aptos initlization, rerun this after starting an aptos node for proper support.

```

Reply:
I verified I ran `npm ci` and waited for the faucet to come up, but I'm still getting this and not able to proceed.

```
$ npm run start

> axelar-examples@1.0.0 start
> node scripts/runStart

Error: ENOENT: no such file or directory, open 'node_modules/@axelar-network/axelar-cgp-aptos/aptos/modules/axelar/build/AxelarFramework/package-metadata.bcs'
    at Object.openSync (node:fs:590:3)
    at Object.readFileSync (node:fs:458:35)
    at AptosNetwork.<anonymous> (/home/stephen/workspace/axelar-examples/node_modules/@axelar-network/axelar-local-dev/dist/aptos/AptosNetwork.js:64:50)
    at Generator.next (<anonymous>)
    at /home/stephen/workspace/axelar-examples/node_modules/@axelar-network/axelar-local-dev/dist/aptos/AptosNetwork.js:8:71
    at new Promise (<anonymous>)
    at __awaiter (/home/stephen/workspace/axelar-examples/node_modules/@axelar-network/axelar-local-dev/dist/aptos/AptosNetwork.js:4:12)
    at AptosNetwork.deploy (/home/stephen/workspace/axelar-examples/node_modules/@axelar-network/axelar-local-dev/dist/aptos/AptosNetwork.js:63:16)
    at AptosNetwork.deployAxelarFrameworkModules (/home/stephen/workspace/axelar-examples/node_modules/@axelar-network/axelar-local-dev/dist/aptos/AptosNetwork.js:96:21)
    at /home/stephen/workspace/axelar-examples/node_modules/@axelar-network/axelar-local-dev/dist/aptos/aptosNetworkUtils.js:27:46 {
  errno: -2,
  syscall: 'open',
  code: 'ENOENT',
  path: 'node_modules/@axelar-network/axelar-cgp-aptos/aptos/modules/axelar/build/AxelarFramework/package-metadata.bcs'
}
Skip Aptos initlization, rerun this after starting an aptos node for proper support.
Creating Moonbeam with a chainId of 2500...
Deploying the ConstAddressDeployer for Moonbeam...
Deployed at 0x69aeB7Dc4f2A86873Dae8D753DE89326Cf90a77a
Deploying the ConstAddressDeployer for Moonbeam...
Deployed at 0x69aeB7Dc4f2A86873Dae8D753DE89326Cf90a77a
Deploying the Axelar Gateway for Moonbeam...
Deployed at 0x9E404e6ff4F2a15C99365Bd6615fCE3FB9E9Cb76
Deploying the Axelar Gas Receiver for Moonbeam...
Deployed at 0x783ce2eF32Aa74B41c8EbbbeC6F632b6Da00C1e9
Deploying the Express Service Contract for Moonbeam...
Deployed ExpressService at 0x5A8b2B50a73E83aAfE73c636ed2D11db9c0e8203
Deploying Axelar Wrapped aUSDC for Moonbeam...
Deployed at 0x5d8C7D94B16a1Ba4865acC6bc039e2877925dC4F
Creating Avalanche with a chainId of 2501...
Deploying the ConstAddressDeployer for Avalanche...
Deployed at 0x69aeB7Dc4f2A86873Dae8D753DE89326Cf90a77a
Deploying the ConstAddressDeployer for Avalanche...
Deployed at 0x69aeB7Dc4f2A86873Dae8D753DE89326Cf90a77a
Deploying the Axelar Gateway for Avalanche...
Deployed at 0x54C156f087EBA12a869E0CBb49548c3F55E80edC
Deploying the Axelar Gas Receiver for Avalanche...
Deployed at 0xD3cB555CB530C3374B122242b15d2943445c31d6
Deploying the Express Service Contract for Avalanche...
Deployed ExpressService at 0xC89c53D6C7031DA523387f72588380900955533D
Deploying Axelar Wrapped aUSDC for Avalanche...
Deployed at 0x19a13F9C3820B148e68E5aEb67cD1BE20c085CD0
Creating Fantom with a chainId of 2502...
Deploying the ConstAddressDeployer for Fantom...
Deployed at 0x69aeB7Dc4f2A86873Dae8D753DE89326Cf90a77a
Deploying the ConstAddressDeployer for Fantom...
Deployed at 0x69aeB7Dc4f2A86873Dae8D753DE89326Cf90a77a
Deploying the Axelar Gateway for Fantom...
Deployed at 0xcb189eB52ca573eFD633d07A3B5357e4d989D743
Deploying the Axelar Gas Receiver for Fantom...
Deployed at 0x85Fa9202C6Be69e9889CC0247Af72ABc70DbF542
Deploying the Express Service Contract for Fantom...
Deployed ExpressService at 0xfFA0EF8701960F8292d3aE5f6AC22C7fA70Da0d6
Deploying Axelar Wrapped aUSDC for Fantom...
Deployed at 0x33251f191aa60F7AA30A5323Eb301557c95f1867
Creating Ethereum with a chainId of 2503...
Deploying the ConstAddressDeployer for Ethereum...
Deployed at 0x69aeB7Dc4f2A86873Dae8D753DE89326Cf90a77a
Deploying the ConstAddressDeployer for Ethereum...
Deployed at 0x69aeB7Dc4f2A86873Dae8D753DE89326Cf90a77a
Deploying the Axelar Gateway for Ethereum...
Deployed at 0x013459EC3E8Aeced878C5C4bFfe126A366cd19E9
Deploying the Axelar Gas Receiver for Ethereum...
Deployed at 0x28f8B50E1Be6152da35e923602a2641491E71Ed8
Deploying the Express Service Contract for Ethereum...
Deployed ExpressService at 0xc918b5BA9127677fDbc39C5e3005047F6565dce6
Deploying Axelar Wrapped aUSDC for Ethereum...
Deployed at 0x38Aa1Cb12E5263eC0c6e9febC25B01116D346CD4
Creating Polygon with a chainId of 2504...
Deploying the ConstAddressDeployer for Polygon...
Deployed at 0x69aeB7Dc4f2A86873Dae8D753DE89326Cf90a77a
Deploying the ConstAddressDeployer for Polygon...
Deployed at 0x69aeB7Dc4f2A86873Dae8D753DE89326Cf90a77a
Deploying the Axelar Gateway for Polygon...
Deployed at 0xc7B788E88BAaB770A6d4936cdcCcd5250E1bbAd8
Deploying the Axelar Gas Receiver for Polygon...
Deployed at 0xC573c722e21eD7fadD38A8f189818433e01Ae466
Deploying the Express Service Contract for Polygon...
Deployed ExpressService at 0x194598519587185673fec0E2C3e9F9Ed94bF2451
Deploying Axelar Wrapped aUSDC for Polygon...
Deployed at 0xC1a50e3611C2c500562803945C0a746c9cFb1939
Serving 5 networks on port 8500
```